src/Dependencies/PooledObjects/PooledDelegates.cs

```diff
@@ -197,6 +198,18 @@ public static Releaser GetPooledAction<TArg>(Action<TArg> unboundAction, TArg ar
         public static Releaser GetPooledFunction<TArg, TResult>(Func<TArg, TResult> unboundFunction, TArg argument, out Func<TResult> boundFunction)
             => GetPooledDelegate<FuncWithBoundArgument<TArg, TResult>, TArg, Func<TArg, TResult>, Func<TResult>>(unboundFunction, argument, out boundFunction);
 
+        /// <summary>
+        /// Equivalent to <see cref="GetPooledFunction{TArg, TResult}(Func{TArg, TResult}, TArg, out Func{TResult})"/>,
+        /// except typed such that it can be used to create a pooled <see cref="ConditionalWeakTable{TKey,
+        /// TValue}.CreateValueCallback"/>.
+        /// </summary>
+        public static Releaser GetPooledCreateValueCallback<TArg, TKey, TValue>(
```

Comment:
><TArg, TKey, TValue> [](http://example.com/codeflow?start=59&length=20)

The order of type parameters feels unconventional. Usually `TArg` is placed last, or immediately before `TResult`.